core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -7463,6 +7518,13 @@ components:
       required: true
       schema:
         type: string
+    googleProjectIdPathParam:
+      name: googleProjectId
+      in: path
+      description: Google Project ID
+      required: true
+      schema:
+        type: string
```

Comment:
will need to coordinate with #3225 on this; see also my comment on that other PR:

https://github.com/broadinstitute/rawls/pull/3225/files#r2006421767